### PR TITLE
create rbac rules for prometheus + bump version

### PR DESCRIPTION
**Pull request by gianrubio**
bump alertmanager, prometheus and operator version

### CODE REVIEW

File: helm/prometheus/values.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ externalUrl: ""
 ##
 image:
   repository: quay.io/prometheus/prometheus
-  tag: v1.5.2
+  tag: v1.7.0
```
Comment (brancz) [inline helm/prometheus/values.yaml]: v1.7.1 has been released

**Comment by brancz** [comment]
as always with helm changes, I'm leaving the last call up to @mgoodness

**Comment by mgoodness** [comment]
Are there any functionality changes we can/should include? Image tags/versions can be specified on the command line (`--set image.tag=...`) or via values file, without bumping chart versions.

**Comment by gianrubio** [comment]
@mgoodness I also add the rbac rules for prometheus. 

Do you think it's a good ideia to add rbac rules for alertmanager? For now it's running with the default `serviceAccount` and most of the case it's a root account.

I thought to add an empty `clusterRole` just to isolate alertmanager in the cluster.

File: helm/prometheus/templates/clusterrolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  name: {{ template "fullname" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "fullname" . }}
+    namespace: {{ .Release.Namespace }}
+{{- end }}
```
Comment (xiang90) [inline helm/prometheus/templates/clusterrolebinding.yaml]: new line?

File: example/rbac/alertmanager/alertmanager-cluster-role.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: alertmanager
+rules:
+- apiGroups: [""]
+  resources:
+  - configmaps
```
Comment (brancz) [inline example/rbac/alertmanager/alertmanager-cluster-role.yaml]: Actually the Alertmanager pods are not accessing the Kubernetes API at all, so there is no need for a ClusterRole at all.Actually the Alertmanager pods are not accessing the Kubernetes API at all, so there is no need for a ClusterRole at all.

Comment (gianrubio) [inline example/rbac/alertmanager/alertmanager-cluster-role.yaml]: Yes, I realised this and removed the rbac for alertmanager.

File: helm/prometheus/templates/clusterrolebinding.yaml (hunk 2 above)
Comment (gianrubio) [inline helm/prometheus/templates/clusterrolebinding.yaml]: Done

**Comment by gianrubio** [comment]
> edit2: alertmanager has a config-reloader pod, so I'll add the right rbac rule.

@mgoodness alertmanager doesn't need any rbac rules, because it didn't talk to the k8s api. 

The PR are ready to merge!

**Comment by mgoodness** [comment]
LGTM! Thanks, and sorry it took so long to merge. Published to the opsgoodness repo.

**Pull request merged**